src/main/java/org/ideoholic/curium/model/periods/dto/Periodmaster.java

```diff
@@ -10,150 +12,58 @@
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
-import javax.persistence.OneToMany;
-
-import static javax.persistence.GenerationType.IDENTITY;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
 import javax.persistence.Table;
 
-import org.ideoholic.curium.model.feescollection.dto.Feescollection;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
 /**
  * Periodmaster generated by hbm2java
  */
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
 @Entity
 @Table(name = "periodmaster")
 public class Periodmaster implements java.io.Serializable {
 
-	private Integer idperiodmaster;
-	private String class_;
-	private Integer totalperiods;
-	private Integer totalbreaks;
-	private String daystart;
-	private String dayend;
-	private String durationofperiod;
-	private String academicyear;
-	private int branchid;
-	private int userid;
-	
-	@OneToMany(cascade = CascadeType.ALL, mappedBy = "Periodmaster")
-	private Set<Perioddetails> periodDetails =
-			new HashSet<Perioddetails>(0);
-	
-	public Periodmaster() {
-	}
-
-	public Periodmaster(String class_, Integer totalperiods,
-			Integer totalbreaks, String daystart, String dayend, int branchid,
-			String durationofperiod, String academicyear, Set<Perioddetails> periodDetails, int userid) {
-		this.class_ = class_;
-		this.totalperiods = totalperiods;
-		this.totalbreaks = totalbreaks;
-		this.daystart = daystart;
-		this.dayend = dayend;
-		this.durationofperiod = durationofperiod;
-		this.academicyear = academicyear;
-		this.periodDetails = periodDetails;
-		this.branchid = branchid;
-		this.userid = userid;
-	}
-
 	@Id
 	@GeneratedValue(strategy = IDENTITY)
 	@Column(name = "idperiodmaster", unique = true, nullable = false)
-	public Integer getIdperiodmaster() {
-		return this.idperiodmaster;
-	}
-
-	public void setIdperiodmaster(Integer idperiodmaster) {
-		this.idperiodmaster = idperiodmaster;
-	}
+	private Integer idperiodmaster;
 
 	@Column(name = "class", length = 45)
-	public String getClass_() {
-		return this.class_;
-	}
-
-	public void setClass_(String class_) {
-		this.class_ = class_;
-	}
+	private String class_;
```

Comment:
```suggestion
	private String class_;
```
@musaibasrar Why is there an underscore appended to this variable?